pkg/rpc/server/server_helper_test.go

```diff
@@ -85,10 +85,10 @@ func initServerWithInMemoryChain(t *testing.T) (*core.Blockchain, *Server, *http
 
 type FeerStub struct{}
 
-func (fs *FeerStub) FeePerByte() util.Fixed8 {
+func (fs *FeerStub) FeePerByte() int64 {
 	return 0
 }
 
-func (fs *FeerStub) GetUtilityTokenBalance(acc util.Uint160) util.Fixed8 {
-	return util.Fixed8FromInt64(1000000)
+func (fs *FeerStub) GetUtilityTokenBalance(acc util.Uint160) int64 {
+	return 100000000000000
```

Comment:
It is worth to have 'native.GASFactor` or smth similar.

Reply:
Fixed by using `native.GASFactor`.